### PR TITLE
Add the capability of remembering the last used room

### PR DESCRIPTION
**Pull request by imobach**

### CODE REVIEW

File: src/app/components/user/jh-signin-form.directive.js [hunk 1]
```diff
@@ -56,6 +56,13 @@
         if (UserService.getUser() !== null) {
           vm.username = UserService.getUser().username;
         }
+
+        if (vm.room === null) {
+          var lastRoomId = UserService.getSetting("lastRoom");
+          vm.room = _.find(rooms, function (room) {
+             return room.room === lastRoomId;
```
Comment (ancorgs) [inline src/app/components/user/jh-signin-form.directive.js]: wouldn't this be room.id? That's how we do it in `StatesService`

Comment (ancorgs) [inline src/app/components/user/jh-signin-form.directive.js]: Oh, `id` is always the same than `room` thanks to `roomFactory`.

**Comment by ancorgs** [comment]
lgtm

**Comment by imobach** [comment]
Thanks!

**Pull request merged**